frontend/src/metabase/setup/components/LicenseTokenStep/LicenseTokenForm.tsx

```diff
@@ -42,17 +63,54 @@ export const LicenseTokenForm = ({
                   setValues({ license_token: trimmed });
                 }
               }}
+              rightSection={
+                <Box>
+                  <HoverCard position="bottom-end" initiallyOpened={true}>
```

Comment:
I was a little surprised to find this opened initially, felt a little buggy to me?

Reply:
Of course 🤦  dev leftover https://github.com/metabase/metabase/pull/59464/commits/d13625f062a8ba9a2810aab1fc018c6ae12ab22d